modules/eslint-plugin/scripts/generate-config.ts

```diff
@@ -94,7 +98,7 @@ function writeConfig(
     plugins: ['@ngrx'],
     rules: configRules,
   };
-  if (configName.includes('all')) {
+  if (requireParserOptions) {
```

Comment:
Another fix: enable parser options for configurations where these are required.
Otherwise these will throw during linting.